meta-oe/recipes-support/driverctl/driverctl_git.bb

```diff
@@ -0,0 +1,26 @@
+SUMMARY = "driverctl is a device driver control utility for Linux"
+LICENSE = "LGPLv2.1"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/LGPL-2.1;md5=1a6d268fd218675ffea8be556788b780"
+
+DEPENDS = "systemd"
+RDEPENDS_${PN} += "bash"
+
+SRC_URI = " git://gitlab.com/driverctl/driverctl.git"
+SRCREV = "fa9dce43d1a667d6e6e26895fbed01b3b04362c9"
+
+S = "${WORKDIR}/git"
+
+inherit pkgconfig
```

Comment:
perhaps inheriting bash-completion is also needed here.

Reply:
Ok, I'll test local build and then update.

Reply:
Added bash-completion.